src/test/v2.0/advanced/fdc3.raiseIntent.ts

```diff
@@ -0,0 +1,119 @@
+import { ChannelError, PrivateChannel } from "fdc3_2_0";
+import { assert, expect } from "chai";
+import { wait } from "../../../utils";
+import { RaiseIntentControl2_0, IntentResultType, IntentApp } from "./intent-support-2.0";
+import constants from "../../../constants";
+
+const control = new RaiseIntentControl2_0();
+
+/**
+ * Details on the mock apps used in these tests can be found in /mock/README.md
+ */
+export default () =>
+  describe("fdc3.raiseIntent", () => {
+    afterEach(async function afterEach() {
+      await control.closeIntentAppWindow(this.currentTest.title);
+    });
+
+    const RaiseIntentSingleResolve = "(2.0-RaiseIntentSingleResolve) Should start app intent-a when raising intent 'aTestingIntent1' with context 'testContextX'";
+    it(RaiseIntentSingleResolve, async () => {
+      await control.listenForError();
+      const result = control.receiveContext("fdc3-intent-a-opened");
+      const intentResolution = await control.raiseIntent("aTestingIntent", "testContextX");
+      control.validateIntentResolution(IntentApp.IntentAppA, intentResolution);
+      await result;
+    });
+
+    const RaiseIntentTargetedAppResolve = "(2.0-RaiseIntentTargetedAppResolve) Should start app intent-a when raising intent 'aTestingIntent1' with context 'testContextX'";
```

Comment:
Definition says intent-a... and you wait for fdc3-intent-a-opened, but the defined test uses IntentAppB:

>- `2.0-RaiseIntentTargetedAppResolve`: Repeat the above test, but:
>  - In the first step use `fdc3.raiseIntent("sharedTestingIntent1", testContextX, {"appID": "<B’s appId>"})` to start app B,

```suggestion
    const RaiseIntentTargetedAppResolve = "(2.0-RaiseIntentTargetedAppResolve) Should start app intent-b when raising intent 'sharedTestingIntent1' with context 'testContextX'";
```

Also fix context you are waiting for to confirm it started